Dockerfile.ci

```diff
@@ -98,8 +98,7 @@ RUN apt-get update \
 # Only copy mysql/mssql installation scripts for now - so that changing the other
```

Comment:
Should we add a frontend definition to this file also?
```
# syntax=docker/dockerfile:1.3
```

Reply:
Why not :). Good point.